Loader/Loader.py

```diff
@@ -33,12 +34,58 @@ def __init__(self, dynamoDBRegion, logLevel):
     self.dynDb = boto3.resource('dynamodb', region_name=dynamoDBRegion)
     self.currentTime = str(datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
     self.workloadSpecType = Loader.WORKLOADSPECTYPE
-
+  #=======================================================================================================================================
+  # DDB Unload, YAML file deletion and copy workload
+  #---------------------------------------------------------------------------------------------------------------------------------------
+  def copy_workload(self):
+      # determine older environment name from source_specname filename passed
+      # determine new environment name to replace old environment name with from file passed.
+
+      sourceWorkloadName = os.path.split(args.sourceWorkloadPath)
+      newWorkloadName = os.path.split(args.newWorkloadPath)
+
+      logger.info("sourceWorkloadName {}\n".format(sourceWorkloadName[1]))
+      logger.info("newWorkloadName {}\n".format(newWorkloadName[1]))
+
+      source = re.search("(?<=Quest-)([A-Z]{2}\d{2})", sourceWorkloadName[1])
+      orig_env = source.group(0)
+      dest = re.search("(?<=Quest-)([A-Z]{2}\d{2})", newWorkloadName[1])
+      new_env = dest.group(0)
```

Comment:
These regex lines are specific to a particular use case.  The product needs remain generic and reusable by other orgs, as it is Open Source